### PR TITLE
Development

### PR DESCRIPTION
**Pull request by NatJensen**
**Sprint 2**

For testing purposes, we have set the distance in MapScreen.js to a very high number so it is possible to test the app without going to the giants.

### CODE REVIEW

File: components/Giant/GiantStyles.js [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+    resizeMode: "cover"
+  },
+  imageText: {
+    margin: 10,
+    fontSize: RFPercentage(5),
+    fontWeight: "bold",
+    color: "#000",
+    fontFamily: "amatic-sc"
+  },
+  ImageTextPlacement: {
```
Comment (ayanlee) [inline components/Giant/GiantStyles.js]: This should be in lower camel case

File: screens/PracticalInfo/PracticalInfo.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { Text, View, ScrollView } from "react-native";
+import styles from "./styles";
```
Comment (ayanlee) [inline screens/PracticalInfo/PracticalInfo.js]: Maybe this should be "Styles" instead?

File: components/Giant/Giant.js [hunk 3]
```diff
@@ -1,33 +1,31 @@
 import React from "react";
-import { Text, View, Image } from "react-native";
-import * as Font from "expo-font";
+import { Text, View, Image, ImageBackground } from "react-native";
 import styles from "./GiantStyles.js";
```
Comment (ayanlee) [inline components/Giant/Giant.js]: I think this should be 'Styles' as well

File: components/Reward/SmallReward.js [hunk 4]
```diff
@@ -1,16 +1,10 @@
-import {
-    View,
-    Text,
-    TouchableOpacity,
-    Image
-} from 'react-native';
+import { Image } from 'react-native';
 import React from 'react';
 import styles from "./styles";
```
Comment (ayanlee) [inline components/Reward/SmallReward.js]: Same thing with naming

File: data/giants.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
+export default [
```
Comment (ayanlee) [inline data/giants.js]: Alternatively, you could've created a .json file.

File: screens/MapScreen/MapScreen.js [hunk 6]
```diff
@@ -1,47 +1,157 @@
...
+    this.setState({
+      date: now
+    });
+  }
+
+  getOrdinalNum(n) {
+    return (
+      n +
+      (n > 0
+        ? ["th", "st", "nd", "rd"][(n > 3 && n < 21) || n % 10 > 3 ? 0 : n % 10]
```
Comment (ayanlee) [inline screens/MapScreen/MapScreen.js]: Maybe you could turn some of these predicates into functions

File: screens/PracticalInfo/PracticalInfo.js (hunk 2 above)
Comment (NatJensen) [reply screens/PracticalInfo/PracticalInfo.js]: Which part do you want to be "Styles"? Is it import styles or is the file name that should be in uppercase?

Comment (ayanlee) [reply screens/PracticalInfo/PracticalInfo.js]: import Styles

File: data/giants.js (hunk 5 above)
Comment (NatJensen) [reply data/giants.js]: json fil created and push to development. Just need to implement it to the code where the data is used.

File: screens/MapScreen/MapScreen.js (hunk 6 above)
Comment (NatJensen) [reply screens/MapScreen/MapScreen.js]: All logic is within this function so we will keep it as it is.

**Pull request merged**